assets/js/app.js

```diff
@@ -578,6 +98,7 @@ liveSocket.connect()
 // >> liveSocket.enableLatencySim(1000)
 window.liveSocket = liveSocket
 
+// REVIEW: I can't find hidePII or showPII used anywhere. Can we remove?
```

Comment:
I use it from the browser JS console during demos to blur out PII 😁 

I know its weird but I couldn't think of a better way

Reply:
What's PII?